package.json

```diff
@@ -1,66 +1,54 @@
 {
-  "name": "ovh-angular-browser-alert",
-  "version": "1.0.0",
-  "description": "Angular component to display an alert when your browser isn't supported",
-  "main": "dist/ovh-angular-browser-alert.min.js",
-  "repository": "ovh-ux/ovh-angular-browser-alert",
-  "author": "OVH SAS",
+  "name": "@ovh-ux/ng-ovh-browser-alert",
+  "version": "0.0.0",
```

Comment:
Why reset the version ? Package is not private

Reply:
I'm targeting a `v1.0.0` release with a BREAKING CHANGE (module name has changed).